protocol/requests/create_community_request.go

```diff
@@ -17,17 +18,19 @@ var (
 )
 
 type CreateCommunity struct {
-	Name        string                               `json:"name"`
-	Description string                               `json:"description"`
-	Color       string                               `json:"color"`
-	Emoji       string                               `json:"emoji"`
-	Membership  protobuf.CommunityPermissions_Access `json:"membership"`
-	EnsOnly     bool                                 `json:"ensOnly"`
-	Image       string                               `json:"image"`
-	ImageAx     int                                  `json:"imageAx"`
-	ImageAy     int                                  `json:"imageAy"`
-	ImageBx     int                                  `json:"imageBx"`
-	ImageBy     int                                  `json:"imageBy"`
+	Name                          string                               `json:"name"`
+	Description                   string                               `json:"description"`
+	Color                         string                               `json:"color"`
+	Emoji                         string                               `json:"emoji"`
+	Membership                    protobuf.CommunityPermissions_Access `json:"membership"`
+	EnsOnly                       bool                                 `json:"ensOnly"`
+	Image                         string                               `json:"image"`
+	ImageAx                       int                                  `json:"imageAx"`
+	ImageAy                       int                                  `json:"imageAy"`
+	ImageBx                       int                                  `json:"imageBx"`
+	ImageBy                       int                                  `json:"imageBy"`
+  MessageArchiveSeedingEnabled  bool                                 `json:"messageArchiveSeedingEnabled,omitempty"`
+  MessageArchiveFetchingEnabled  bool                                 `json:"messageArchiveFetchingEnabled,omitempty"`
```

Comment:
I made those two `omitempty` so we can potentially put these changes into production without having the need for desktop and/or mobile to do anything in particular.